modules/mobile/docs/openapi.yaml

```diff
@@ -314,7 +314,7 @@ paths:
           $ref: '#/components/responses/504'
       security:
         - Bearer: []
-      summary: /v0/appointments/check-in (DRAFT)
+      summary: /v0/appointments/check-in
```

Comment:
@cadibemma can you change this to `summary: /v0/appointments/check-in (DRAFT)`? I know that's from my code but I just need the docs to be back to draft as this is in flux at the moment. Otherwise lgtm

Reply:
updated